modules/manage/partials/tiered-storage.adoc

```diff
@@ -1329,6 +1329,47 @@ ifndef::env-kubernetes[Edit them with the `rpk cluster config edit` command.]
 | The public certificate used to validate the TLS connection to object storage. If this is empty, Redpanda uses your operating system's CA cert pool.
 |===
 
+== Faster partition movement during cluster resize
+
+Tiered Storage gives you the option to boost node operations particularly when resizing a cluster. For instance, adding a new node to increase capacity for an overburdened cluster could introduce additional stress as partition replicas are transferred to the new node. For cloud deployments, commissioning or decommissioning a broker can be a slow and expensive process, especially across multiple availability zones (AZ).
+
+Tiered Storage can help make the cluster resize process more cost-efficient by leveraging data that has already been uploaded to object storage. Instead of transferring all data from local storage to the reassigned replicas, the replicas can instead be initialized using object storage.
+
+To accomplish this, Redpanda takes an on-demand snapshot of a log segment that contains an offset already uploaded to object storage. This is a later offset compared to the default log start offset managed by the glossterm:Raft[] protocol. When an empty replica is initialized, the later offset uploaded to object storage can be used as the start offset source. Reads that access data before the offset in object storage can be executed as remote reads. Therefore, only the locally-retained data after the uploaded offset needs to be sent over the network to the replicas. This also results in less time taken overall to reconfigure the cluster, and faster recovery in case of failures.    
+
+=== Configure partition movement through Tiered Storage
+
+In addition to configuring topics for Tiered Storage, also configure at least one of the following cluster configuration properties:
```

Comment:
'to activate faster partition movement when nodes enter and leave the cluster, In addition to configuring topics for Tiered Storage, also configure at least one of the following cluster configuration properties to limit the size of data replicated across nodes to local storage using Raft'